components/automate-ui/src/app/modules/infra-proxy/org-edit/org-edit.component.html

```diff
@@ -30,6 +30,12 @@
         formControlName="admin_key" data-cy="update-org-admin-key"></textarea>   
       </label>
     </chef-form-field>
+    <chef-form-field id='projects-container'>
+      <app-projects-dropdown
```

Comment:
The width of the form on this page is too narrow. 
It needs to be at least as wide as on the teams page shown to remove the excess line break--compare (2) on the teams page to (1) on this page.
![image](https://user-images.githubusercontent.com/6817500/82715658-0940aa80-9c49-11ea-93c7-01ccd7ed6e82.png)

Reply:
@msorens added some width which is equal to teams details tab view width. :+1: